src/main/joyride/sci.cljs

```diff
@@ -19,7 +19,8 @@
 (def !ctx (volatile!
            (sci/init {:classes {'js goog/global
                                 :allow :all}
-                      :namespaces (:namespaces pconfig/config)
+                      :namespaces (merge (:namespaces pconfig/config)
```

Comment:
Minor thing but for performance reasons it's better to just use `assoc` here.

Reply:
Tried, but I can't figure out how to do this with `assoc` while keeping the merge order of `joyride.core` las...

Reply:
`(assoc (:namespaces ...) 'joyride.core {...})`

Reply:
Ah, of course. 🤦